include/flexflow/model.h

```diff
@@ -246,29 +246,43 @@ class NoOp;
 ParallelConfig get_basic_data_parallel_config(int num_parts, int dims);
 
 class Cast;
+class CastParams;
 class Concat;
```

Comment:
Hey, why are all of these `Param` forward declarations here? Aren't they taken care of by `#include`ing `operator_params.h`?

Reply:
I think a few of these were there before so I followed that pattern, but could we put `#include operator_params.h` in `model.cc`? I think that might be faster at compile time.